### PR TITLE
Feat/linear damping

### PR DESCRIPTION
**Pull request by ishiko732**
- https://github.com/open-spaced-repetition/fsrs-rs/pull/239

### CODE REVIEW

**Comment by L-M-Sherlock** [comment]
Please wait for the stable release of Anki and I will update the optimizer.

File: src/fsrs/algorithm.ts [hunk 1]
```diff
@@ -159,16 +158,25 @@ export class FSRSAlgorithm {
     return this.apply_fuzz(newInterval, elapsed_days)
   }
 
+  /**
+   * @see https://github.com/open-spaced-repetition/fsrs4anki/issues/697
+   */
+  linear_damping(delta_d: number, old_d: number): number {
+    return +((delta_d * (10 - old_d)) / 9).toFixed(8)
```
Comment (L-M-Sherlock) [inline src/fsrs/algorithm.ts]: Remove unnecessary parentheses.

```suggestion
    return +(delta_d * (10 - old_d) / 9).toFixed(8)
```

**Comment by ishiko732** [comment]
> Please wait for the stable release of Anki and I will update the optimizer.

Anki has released a new version. https://github.com/ankitects/anki/releases/tag/24.11

**Review by L-M-Sherlock — APPROVED** [review]
LGTM

**Pull request merged**